nvtabular/io.py

```diff
@@ -58,6 +58,18 @@
 #
 
 
+def _check_shuffle_arg(shuffle):
+    if shuffle not in ("per-chunk", "per-worker", "full", None):
+        if shuffle is True:
+            shuffle = "per-worker"
+            warnings.warn('`shuffle=True` is deprecated. Using `shuffle="per-worker".`')
+        elif shuffle is False:
+            shuffle = None
+        elif shuffle == "full":
+            raise ValueError('`shuffle="full"` is not yet supported.')
```

Comment:
Should we also add a catch-all here for when the user passes something else? Right now if you pass say ```shuffle=1``` it will I guess work the same as 'per-chunk' - but its not at all clear.  Maybe something like 
```suggestion
            raise ValueError('`shuffle="full"` is not yet supported.')
        else:
            raise ValueError(f"shuffle='{shuffle}' isn't a valid shuffling type")
```

Reply:
Ah - My bad.  Thanks for catching this :)